raiden/tests/integration/fixtures/raiden_network.py

```diff
@@ -219,21 +221,29 @@ def raiden_network(
         channel_settle_timeout=settle_timeout,
     )
 
-    exception = RuntimeError("`raiden_network` fixture setup failed, nodes are unreachable")
-    with gevent.Timeout(seconds=timeout(blockchain_type), exception=exception):
-        wait_for_channels(
-            app_channels=app_channels,
-            payment_network_address=blockchain_services.deploy_registry.address,
-            token_addresses=token_addresses,
-            deposit=deposit,
+    if start_raiden_apps:
+        exception = RuntimeError("`raiden_network` fixture setup failed, nodes are unreachable")
+        with gevent.Timeout(seconds=timeout(blockchain_type), exception=exception):
+            wait_for_channels(
+                app_channels=app_channels,
+                payment_network_address=blockchain_services.deploy_registry.address,
+                token_addresses=token_addresses,
+                deposit=deposit,
+            )
+
+        # Force blocknumber update
+        exception = RuntimeError("Alarm failed to start and set up start_block correctly")
+
+        with gevent.Timeout(seconds=5, exception=exception):
+            wait_for_alarm_start(raiden_apps)
+    else:
+        # simply wait for another confirmation blocks number since we can't
+        # confirm channels have been made through the raiden apps that are not started
```

Comment:
Why is this necessary?

Reply:
Because there is no other way to wait until the channels ae created a sis done above when apps are actually started since the raiden apps are empty

Reply:
> Because there is no other way to wait until the channels ae created

The proxies will poll on the transaction hash and wait for the transaction to be confirmed. Is there anything else that has to be done?

Reply:
there are no proxes as raiden has not yet started.